033-OSSDatabaseMigration/Coach/07-private-endpoint.md

```diff
@@ -0,0 +1,8 @@
+# Challenge 7: Private Endpoint
+
+[< Previous Challenge](./06-online-cutover-validation.md) - **[Home](./README.md)**  - [Next Challenge >](./08-replication.md)
+
+## Coach Tips
+
+1) If the attendee has trouble with private endpoint working with the Pizzeria application, make sure they check the logs. It's common for the dataSourceURL/username/password to be misconfigured
+2) This challenge will not work if the attendee is using the Basic tier: https://docs.microsoft.com/en-us/azure/mysql/concepts-data-access-security-private-link
```

Comment:
Make the URL a sub bullet of #2 and wrap it in angle brackets, eg: `<https://docs.microsoft.com/en-us/azure/mysql/concepts-data-access-security-private-link>`

Reply:
Fixed